robot_specific_config/abb_irb1300_10_115_moveit_config/config/moveit_controllers.yaml

```diff
@@ -2,20 +2,22 @@ trajectory_execution:
   allowed_execution_duration_scaling: 1.2
   allowed_goal_duration_margin: 0.5
   allowed_start_tolerance: 0.01
+  trajectory_duration_monitoring: true
 
 moveit_controller_manager: moveit_simple_controller_manager/MoveItSimpleControllerManager
 
-controller_names:
-  - joint_trajectory_controller
+moveit_simple_controller_manager:
+  controller_names:
+    - joint_trajectory_controller
 
-joint_trajectory_controller:
-  action_ns: follow_joint_trajectory
-  type: FollowJointTrajectory
-  default: true
-  joints:
-    - joint_1
-    - joint_2
-    - joint_3
-    - joint_4
-    - joint_5
-    - joint_6
+  joint_trajectory_controller:
+    action_ns: follow_joint_trajectory
+    type: FollowJointTrajectory
+    default: true
+    joints:
+      - joint_1
+      - joint_2
+      - joint_3
+      - joint_4
+      - joint_5
+      - joint_6
```

Comment:
This change is the same as upstream https://github.com/PickNikRobotics/abb_ros2/pull/60, note that if we don't merge `rolling` the `abb_irb1200_5_90_moveit_config` package will still be broken, but I didn't want to merge upstream since it added _a lot_ to this diff.